src/logic/map_objects/tribes/ship.cc

```diff
@@ -649,6 +649,7 @@ void Ship::ship_update_idle(Game& game, Bob::State& state) {
 					} else {
 						wq->set_filled(wq->get_filled() + 1);
 					}
+					ware->remove(game);
```

Comment:
If this is the fix, please explain this line and that ship_update_idle deserves some refactoring ...